### PR TITLE
Sdk update for ios and android

### PR DESCRIPTION
**Pull request by Javasamurai**

### CODE REVIEW

**Review by vivekb91 — APPROVED** [review]

**Comment by rampr** [comment]
Are notifications going out. Is data about notifications being stored correctly ?

**Comment by Javasamurai** [comment]
@rampr @bijoshtj Don't merge this now.xCode has some issues.

**Comment by Javasamurai** [comment]
@rampr Noticiations are getting shown and click are getting registered in db.

File: ios/config.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   ],
   "frameworks": [
     "SystemConfiguration",
+    "UserNotifications.framework",
```
Comment (rampr) [inline ios/config.json]: you don't need .framework for System frameworks

**Review by rampr — APPROVED** [review]

**Pull request merged**